Fixed the issue of  causes ``KeyError`` when using the parameter ``--import-mode=importlib`` in pytest>=8.2 .
